proposed/0000-safecoin-market-price.md

```diff
@@ -0,0 +1,74 @@
+- Feature Name: SafeCoin_Market_Price
+- Type: New feature
+- Related components: maidsafe_vault, maidsafe_client, maidsafe_types
+- Start Date: 19-07-2015
+- RFC PR: (leave this empty)
+- Issue number: (leave this empty)
+
+# Summary
+
+This RFC is based on the concerns I’ve read in the MaidSafe forum for the last six months. These are some of them:
+
+* https://forum.safenetwork.io/t/lowest-put-price-discovery-and-new-safecoin-issuance/3819
+* https://forum.safenetwork.io/t/cost-of-storing-data-on-safe/4285
+* https://forum.safenetwork.io/t/the-price-of-safecoin-the-economics-behind-it/844
+
+This RFC is only another point of view (maybe not doable, feel free to reject this RFC) that breaks a lot the main design. Maybe this RFC can be decomposed in several RFC’s, smaller and feasible.
+
+
+# Motivation
+
+My first concern is how much cost to store a chunk of data (cost in terms of USD,EUR or BTC). No one is able to answer this question accurately. I’m thinking in a true Offer/Demand market, where you establish how much you want to pay to store data, and in the other side, the vaults establish how much they want to get paid to store that data.
+
+Another important aspect is the idea of pay once and store data forever. I can’t see how can be sustainable in the future. I’d like to see a system where you Put data into the Net as a renting service, like the Amazon Web Services.
+
```

Comment:
Yes this is another area questioned a lot. The notion is farmers must farm *above* the existing amount of data to get rewarded. 

The network has no idea of time and cannot recognise people, certainly not via a payment.  It could with safecoin (perhaps) but then it would need to link a persons data in an account (security issues) which the network holds and can recognise their data.

The network also does not know the notion of time in any sense, it does not use time anywhere except for very local calculations (never transmitted). Time is alien to the network protocol. 

For safety therefor all data is stored forever, unless signed by people.

Signed data is another mechanism being introduced at the moment (RFC in progress) where immutable data can be deleted. This will be more expensive that normal (de-duplicatable data) but can be deleted (by owner).  The owner is basically the private key holder of an embedded public key and will not leak any info about actual identities. 

Another area not covered yet is archive nodes, these will take older data and make them less stressful on the network by creating superchunks, these superchunks will contain millions of chunks in a structure that allows easy (but slower) retrieval of such data. The archive nodes will be paid at a higher rate we imagine that normal farmer nodes. RFC to come though. Dev bundle 3  first :-)